xls/codegen/vast/vast.cc

```diff
@@ -357,6 +359,10 @@ std::string SanitizeIdentifier(std::string_view name) {
       sanitized[i] = '_';
     }
   }
+  if ((system_verilog ? SystemVerilogKeywords() : VerilogKeywords())
```

Comment:
Maybe just add a comment to note that our policy is to put an underscore suffix on any keyword collisions? (Similar to the explainer comment above)

Reply:
done.